src/bhost_fpga.py

```diff
@@ -22,19 +22,10 @@
 
 class FpgaBHost(FpgaHost):
```

Comment:
So Nielan would rather use Object in place of FpgaHost. The object passed would be x-engine.